packages/web/src/common/store/premiumContent/sagas.ts

```diff
@@ -170,7 +170,7 @@ function* updateNewPremiumContentSignatures({
  * Halts if not all nfts have been fetched yet. Similarly, does not proceed if no tracks are in the cache yet.
  * Skips tracks whose signatures have already been previously obtained.
  */
-function* updateNFTGatedTrackAccess(
+async function* updateNFTGatedTrackAccess(
```

Comment:
same here?

Reply:
Great catches, I was doing an await to check types cause it's better than call, and forgot to change back